lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -242,6 +241,10 @@ def output_exclude_path(output_buffer, exclude_path, parent)
         regexp = exclude_path
         output_buffer.puts "    - !ruby/regexp /#{regexp.source}/"
       end
+
+      def safe_autocorrect?(config)
+        config&.fetch('Safe', true) && config&.fetch('SafeAutoCorrect', true)
```

Comment:
Sorry my example didn't take into account the case where config is nil. But wouldn't using save_navigation_operator here have the opposite result of what originally intended?

I'm not sure whether there is a real-world case that config is nil though...

```rb
config = {}
config&.fetch('Safe', true) && config&.fetch('SafeAutoCorrect', true) # => true

config = nil
config&.fetch('Safe', true) && config&.fetch('SafeAutoCorrect', true) # => nil
```

Reply:
Indeed. I fixed it to take that into account as well. WDYT?

Reply:
Looks good to me, thank you!
At first I thought it could be written like following, but yeah, this would invert the meaning when unsafe side.

Maybe the maintainers can give you a better pointer.

```rb
def safe_autocorrect?(config)
  return true if config.nil?

  config.fetch('Safe', true) && config.fetch('SafeAutoCorrect', true)
end
```

Reply:
Thanks for taking the time to brush up on yours! I appreciate it.